compileopts/config.go

```diff
@@ -457,7 +457,14 @@ func (c *Config) OpenOCDConfiguration() (args []string, err error) {
 		args = append(args, "-c", cmd)
 	}
 	if c.Target.OpenOCDTransport != "" {
-		args = append(args, "-c", "transport select "+c.Target.OpenOCDTransport)
+		transport := c.Target.OpenOCDTransport
+		if transport == "swd" {
```

Comment:
In the future, when a transport other than swd is added, a bug will occur that is hard to detect, so we will check if it is swd or not.

Reply:
Thanks, that's a good idea.